catalog/core/catalog-core-api-impl/src/main/java/org/codice/alliance/catalog/core/api/impl/types/IsrAttributes.java

```diff
@@ -231,6 +231,24 @@
                 true /* tokenized */,
                 false /* multivalued */,
                 BasicTypes.STRING_TYPE));
+        DESCRIPTORS.add(new AttributeDescriptorImpl(SNOW_COVER,
+                true /* indexed */,
+                true /* stored */,
+                false /* tokenized */,
+                false /* multivalued */,
+                BasicTypes.BOOLEAN_TYPE));
+        DESCRIPTORS.add(new AttributeDescriptorImpl(SNOW_DEPTH_MIN_CENTIMETERS,
+                true /* indexed */,
+                true /* stored */,
+                false /* tokenized */,
+                false /* multivalued */,
+                BasicTypes.DOUBLE_TYPE));
```

Comment:
I decided to use double because it's higher precision. But I can change it to float if enough reviewers agree.